resources/prosody-plugins/mod_reservations.lua

```diff
@@ -546,29 +546,25 @@ end);
 
 --- Forget reservation details once room destroyed so query is repeated if room re-created
 local function room_destroyed(event)
-    local res;
     local room = event.room
 
     if not is_healthcheck_room(room.jid) then
-        res = reservations[room.jid]
 
         -- drop reservation data for this room
         reservations[room.jid] = nil
 
-        if res then  -- just in case event triggered more than once?
-            module:log("info", "Dropped reservation data for destroyed room %s", room.jid);
+        module:log("info", "Dropped reservation data for destroyed room %s", room.jid);
 
-            local conflict_id = res.meta.conflict_id
-            if conflict_id then
-                local url = api_prefix..'/conference/'..conflict_id;
-                local http_options = {
-                    method = 'DELETE';
-                    headers = http_headers;
-                }
+        local conflict_id = room.jid -- This is same as reservations[room.jid].meta.conflict_id + @your.domain
```

Comment:
Isn't `conflict_id` is meant to be returned from API call during create? 

Assuming that this always matches `room.jid` would break compatibility with specs from the original Reservations module in Jicofo.